fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -69,6 +69,18 @@
     FedoraResource getContainer();
 
     /**
+     * Get the TimeMap/LDPCv of this resource
+     * @return the container for TimeMap/LDPCv of this resource
+     */
+    FedoraResource getLDPCv();
+
+    /**
+     * Create the TimeMap/LDPCv of this resource if it doesn't exist
+     * @return the container for TimeMap/LDPCv of this resource
+     */
+    FedoraResource findOrCreateLDPCv();
```

Comment:
It would likely be more clear to change the name of this method to `findOrCreateTimeMap()`;